warehouse/packaging/models.py

```diff
@@ -874,3 +888,29 @@ class ProhibitedProjectName(db.Model):
     )
     prohibited_by: Mapped[User] = orm.relationship()
     comment: Mapped[str] = mapped_column(server_default="")
+
+
+class ProjectMacaroonWarningAssociation(db.Model):
```

Comment:
We can keep the `__tablename__`, but I don't think any of the other assoc tables in Warehouse include `Association` in the class name 🙂 

```suggestion
class ProjectMacaroonWarnings(db.Model):
```

Reply:
There's `OIDCProviderProjectAssociation` and `ObserverAssociation`

Reply:
Ah, whoops. Let's leave it then!